website/static/font-awesome/css/font-awesome.css

```diff
@@ -0,0 +1,2199 @@
+/*!
```

Comment:
This file shouldn't need to be here; we already have `fa` icons and css classes.